securityscanutils/issuewriter/local_writer.go

```diff
@@ -0,0 +1,47 @@
+package issuewriter
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/google/go-github/v32/github"
+)
+
+type LocalIssueWriter struct {
```

Comment:
nit: Since this is exported, I think a comment clarifying what a local_writer is, and how it is intended to be used, would be valuable